### PR TITLE
Improving JModuleHelper::getModule() test

### PR DESCRIPTION
**Pull request by Hackwar**
See title

### CODE REVIEW

File: tests/unit/suites/libraries/cms/module/JModuleHelperTest.php [hunk 1]
```diff
@@ -83,6 +83,36 @@ public function testGetModule()
...
+
+		$this->assertEquals(
+			$module,
+			null,
+			'There should be no module named false'
+		);
+
+		$module = JModuleHelper::getModule('mod_false');
+
+		$this->assertTrue(is_object($module), 'No object was returned');
```
Comment (wilsonge) [inline tests/unit/suites/libraries/cms/module/JModuleHelperTest.php]: Would it not be better to use assertInternalType for this?

**Comment by Hackwar** [comment]
Updated to use assertInternalType like @wilsonge proposed.

**Comment by spignataro** [comment]
FYI - this test has some incorrect language. 

There anonymous module should have no id

should be:

The anonymous module should have no id


and

There anonymous module should have the given name

should be

The anonymous module should have the given name

**Comment by zero-24** [comment]
@spignataro can you send a PR to fix them? https://docs.joomla.org/Using_the_Github_UI_to_Make_Pull_Requests

**Comment by spignataro** [comment]
Done: https://github.com/joomla/joomla-cms/pull/6572

**Pull request merged**